sdk/core/azure-core/src/test/java/com/azure/android/core/internal/util/serializer/ByteArraySerializerTest.java

```diff
@@ -0,0 +1,40 @@
+package com.azure.android.core.internal.util.serializer;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class ByteArraySerializerTest {
+
+    @Test
+    public void test_getModule() {
+        SimpleModule module = ByteArraySerializer.getModule();
+
+        assertNotNull(module);
+    }
+
+    @Test
+    public void test_serialize() throws IOException {
+        Byte[] byteObjects = new Byte[] { 84, 101, 115, 116, 32, 115, 116, 114, 105, 110, 103 };
```

Comment:
Mind putting a comment on the `byte` arrays that mentions their UTF-8 string representation? It'll make the tests more understandable, right now I can't really make heads or tails on whether this is correct since it is just sequences of numbers.

Reply:
Sure thing.